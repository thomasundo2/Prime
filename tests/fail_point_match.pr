int main()
{
    pt x;
    printpt([192943924239l, 2] & [(1l,2l) : 3l]);
    return 0;
}

