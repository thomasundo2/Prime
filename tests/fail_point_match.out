Fatal error: exception Failure("points must have Lint coordinates")
